test/jb/Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG IMAGE=containerbase/buildpack
+FROM ${IMAGE} as build
+
+RUN touch /.dummy
+
+# renovate: datasource=github-releases lookupName=jsonnet-bundler/jsonnet-bundler
+RUN install-tool jb v0.4.0
```

Comment:
Please move after user, so we can test install without root.

Alternative you can add a second test for it.